src/python/grpcio/grpc/__init__.py

```diff
@@ -1447,6 +1447,19 @@ def stop(self, grace):
         """
         raise NotImplementedError()
 
+    def wait_for_termination(self):
+        """Block current thread until the server stops.
+
+        This is an EXPERIMENTAL API.
+
+        The wait will not consume computational resources during blocking, and it
+        will block indefinitely. There are two ways to unblock:
```

Comment:
Nit: It's a bit contradictory to say it will block indefinitely and then to define to conditions under which it will stop blocking. How about "...and it will block until one of the two following conditions are met:"

Reply:
Thank you for the advice. This is much better!